### DISCUSSION TITLE
Supported Sony Alpha Cameras

### DISCUSSION THREAD

**Discussion by izzyzizou** [Staacks/alpharemote#6] [Q&A]
Is there a list of supported Sony Alpha Cameras?

**Comment by Staacks** [comment]
There is a list on the Play Store and yesterday I added it to this repo's [Readme](https://github.com/Staacks/alpharemote?tab=readme-ov-file#compatibility).

Basically, it should work with any camera that is supported by Sony's physical Bluetooth remote and the list is based on Sony's compatibility info for their remote. At the moment the app is incompatible with new devices release around 2022, which is a bug in version 0.16 - the version with the fix is currently waiting for the review process on Google Play and should become available very soon.
